src/main/java/paymelah/model/debt/DebtList.java

```diff
@@ -91,7 +100,9 @@ public List<Debt> asList() {
      */
     public static DebtList fromList(List<Debt> list) {
         DebtList debtList = new DebtList();
-        debtList.debts.addAll(list);
+        for (Debt debt : list) {
+            debtList = debtList.addDebt(debt);
```

Comment:
Should we keep the old method instead, since this will unnecessarily make new `DebtList`s for each `Debt` in the list?